srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
```

Comment:
The repo name being `nagioscore`, it should probably be the package name as well.

Reply:
With name after tarball, `nagios`, default update-check and wrksrc do work.

Reply:
will change

Reply:
This one should be kept the same. Use `nagios`.

Reply:
Please leave as is, we prefer tarball name for reasons above, additionally for this package `nagios` it is more expected name, only distro calling it nagios-core is gentoo.

Reply:
:+1: will leave as nagios